infer_structcol/run_structcol.py

```diff
@@ -7,10 +7,16 @@
 from structcol import montecarlo as mc
 import structcol.refractive_index as ri
 from infer_structcol import main 
+import matplotlib
+from matplotlib import pyplot as plt
+import seaborn as sns
+import os
+sns.set(font_scale=1.3) 
 
-def calc_refl_trans(volume_fraction, radius, thickness, Sample, ntrajectories=600, nevents=200, seed=None):
+def calc_sigma(volume_fraction, radius, thickness, Sample, ntrajectories, nevents, run_num=100, plot=False, seed=None):
```

Comment:
It might be better to have ``plot=True`` be default, so that users don't need to specifically ask for it to see plots. It's easy to have internal functions (eg. calc_mcmc) explicitly call this function with ``plot=False``.